src/shapes.jl

```diff
@@ -348,7 +348,7 @@ Luxor.Point(4.0, -9.797174393178826e-16)
 whereas
 
 ```
-ngon(0, 0, 4, 4, 0, :close) # draws a polygon
+ngon(0, 0, 4, 4, 0, :close) # draws a polygon
```

Comment:
Sorry if this [non-breaking space](https://en.wikipedia.org/wiki/Non-breaking_space) is deliberate. I just thought this can be replaced with normal space.
```julia
julia> collect("# dra")
5-element Vector{Char}:
 '#': ASCII/Unicode U+0023 (category Po: Punctuation, other)
 ' ': Unicode U+00A0 (category Zs: Separator, space)
 'd': ASCII/Unicode U+0064 (category Ll: Letter, lowercase)
 'r': ASCII/Unicode U+0072 (category Ll: Letter, lowercase)
 'a': ASCII/Unicode U+0061 (category Ll: Letter, lowercase)

julia> collect("# dra")
5-element Vector{Char}:
 '#': ASCII/Unicode U+0023 (category Po: Punctuation, other)
 ' ': ASCII/Unicode U+0020 (category Zs: Separator, space)
 'd': ASCII/Unicode U+0064 (category Ll: Letter, lowercase)
 'r': ASCII/Unicode U+0072 (category Ll: Letter, lowercase)
 'a': ASCII/Unicode U+0061 (category Ll: Letter, lowercase)
```

Reply:
Ah, I also found non-breaking spaces in other .jl files, but these seem not unified and can be replaced with normal spaces.

Reply:
@cormullion Can I make another PR to replace the spaces?